lib/data_migrations/write_benefits_intake_uuids_to_form_submission_attempts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module DataMigrations
+  module WriteBenefitsIntakeUuidsToFormSubmissionAttempts
+    module_function
+
+    def run
+      FormSubmissionAttempt.all.find_each do |attempt|
```

Comment:
At first I wrote `.all.each` here but my linter wanted me to use `.find_each`. Does anyone know why/if it matters?

Reply:
`.find_each` uses batches under the hood, whereas `.each` loads all the submissions at once. There are a lot of FormSubmissionAttempts, so `.find_each` makes sense to me

Reply:
You can also call `find_each` directly on the model without using `all`, which avoids creating an intermediate array of all records before processing them in batches.

Reply:
Fixed! Thanks @TaiWilkin and @freeheeling !